Python-packages/covidcast-py/covidcast/geography.py

```diff
@@ -128,6 +133,33 @@ def name_to_abbr(name: Union[str, Iterable],
     return _lookup(name, STATE_CENSUS.NAME, STATE_CENSUS.ABBR, ignore_case, fixed, ties_method)
 
 
+def fips_to_abbr(code: Union[str, Iterable],
+                 ignore_case: bool = False,
+                 fixed: bool = False,
+                 ties_method: str = "first") -> list:
+    """Look up state abbreviation by FIPS codes with regular expression support.
+
+    Given an individual or list of FIPS codes or regular expressions, look up the corresponding
+    state abbreviation
```

Comment:
```suggestion
    state abbreviation.
```

Reply:
Do you want to give an example? Does this work with 2-digit or 5-digit FIPS?

Reply:
Both, 
```
In [2]: covidcast.geography.fips_to_abbr("12")
Out[2]: ['FL']

In [3]: covidcast.geography.fips_to_abbr("12000")
Out[3]: ['FL']
```
I'll add a note in the docs